classification/views/exports/classification_export_formatter_lab_compare.py

```diff
@@ -13,10 +15,15 @@
 
 class ClassificationlabCompareRow(ExportRow):
 
-    def __init__(self,  AlleleData, lab1, lab2):
-        self.AlleleData = AlleleData
+    def __init__(self,  allele_data: AlleleData, lab1, lab2, comment=None, authorised_date=None, created_date=None,
```

Comment:
Sorry to be pedantic but just want to get into the habbit, would be nice to see
```
lab_1: Lab, lab_2: Lab, comment: Optional[str]=None, authrorised_date: Optional[str]=None, created_date: Optional[str]=None, patient_id: Optional[str]=None
```
as good practise